docs/kyma/04-08-update.md

```diff
@@ -36,67 +36,55 @@ In case of dependency conflicts or major changes between components versions, so
+- If you add a new component to your Kyma deployment, add a top-level Helm chart for that component. Additionally, download the current [Installation custom resource](#custom-resource-installation) on the cluster and add the new component to the components list:
 
    ```bash
-   kubectl -n default edit installation kyma-installation
+   kubectl -n default get installation kyma-installation -o yaml > installation.yaml
    ```
 
-- If you introduced changes in overrides, update the existing ConfigMaps and Secrets. Add new ConfigMaps and Secrets if required. See the [configuration document](#configuration-helm-overrides-for-kyma-installation) for more information on overrides.
+- If you introduce changes in the overrides, create a file with your changes as ConfigMaps or Secrets. See the [configuration document](#configuration-helm-overrides-for-kyma-installation) for more information on overrides.
 
 ## Perform the update
 
-If your changes involve any modifications in the `/resources` folder that includes component chart configurations, perform the whole update process that includes updating the Kyma Installer and triggering the update. If you only modify installation artifacts, for example by adding or removing components in the installation files or adding or removing overrides in the configuration files, only trigger the update process.
+If your changes involve any modifications in the `/resources` folder that includes component chart configurations, perform the steps on "Update with resources modifications". If you only modify installation artifacts, for example by adding or removing components in the installation files or adding or removing overrides in the configuration files, perform the steps on "Update without resources modifications".
 
 Read about each update step in the following sections.
 
-### Update the Kyma Installer on a local deployment
+<div tabs name="perform-the-update">
+   <details>
+   <summary label="update-with-resources-moditifations">
+   Update with resources modifications
+   </summary>
 
-1. Build a new image for the Kyma Installer:
+   1. Check which version you're currently running. Run this command:
 
-   ```bash
-   ./installation/scripts/build-kyma-installer.sh
-   ```
-
-   > **NOTE:** If you started Kyma with the `run.sh` script with a `--vm-driver {value}` parameter, provide the same parameter to the `build-kyma-installer.sh` script.
+      ```bash
+      kyma version
+      ```
 
-2. Restart the Kyma Installer Pod:
+   2. Provide the same version of the current cluster to the upgrade command. Provide also an image name and a tag so that Kyma CLI will build a Docker image with your local changes and push it to the registry. It will also trigger the update process. If you have changes for the overrides or the components list, you can also pass them using `-o` and `-c` flags.
```

Comment:
```suggestion
   2. Provide the same version of the current cluster to the upgrade command. Provide also an image name and a tag so that Kyma CLI will build a Docker image with your local changes and push it to the registry. It will also trigger the update process. If you have changes for the overrides or the components list, you can also pass them using the `-o` and `-c` flags.
```